app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Pet;
+use Illuminate\Http\Request;
+
+class PetController extends Controller
+{
+    public function show() {
+        return view('pets', [
+            'pets' => self::getPet(),
+        ]);
+        return null;
+    }
+
+    public static function getPet() {
```

Comment:
this can be private scope since it's only called within the class definition. also I'd name it `getPets` if returning an array